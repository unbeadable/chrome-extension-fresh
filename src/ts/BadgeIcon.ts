export class BadgeIcon {
    readonly element: HTMLElement = document.createElement('img');

    constructor() {
        this.element.setAttribute(
            'src',
            'https://s3.eu-central-1.amazonaws.com/unbeadable/baseline-report-24px.svg'
        );
        this.element.setAttribute('class', 'unbdbl-badge-icon');
    }
}
