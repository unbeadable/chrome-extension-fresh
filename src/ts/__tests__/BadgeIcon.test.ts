import { BadgeIcon } from '../BadgeIcon';

describe('BadgeIcon', () => {
    it('should match snapshot', () => {
        expect(new BadgeIcon().element).toMatchSnapshot();
    });
});
