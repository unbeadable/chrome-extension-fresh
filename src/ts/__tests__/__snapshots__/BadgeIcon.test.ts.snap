// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadgeIcon should match snapshot 1`] = `
<img
  class="unbdbl-badge-icon"
  src="https://s3.eu-central-1.amazonaws.com/unbeadable/baseline-report-24px.svg"
/>
`;
